GTBitsOfGood/bgc-safety
# Merge spring 2021 final sprint

### Description
- Merging `navbar-style` branch to master caused deployment issue
- Fixed the error (accidentally deleted useEffect call) and added minor changes

### How to Test
1. Run the environemnt locally
2. Log in using admin@bgcma.org credential
3. Check that SwipableDrawer navbar is visible on Home page after logging in

### Important Changes

- Added `authorizationUrl` to nextauth config to prevent auto-login

### Related Github Issues

- none

### Related PRs

- none

### Checklist
